torchmultimodal/models/video_vqvae.py

```diff
@@ -14,47 +14,60 @@
 from torchmultimodal.utils.assertion import assert_equal_lengths
 from torchmultimodal.utils.common import to_tuple_tuple
 
-DEFAULT_ENCODER_IN_CHANNEL_DIMS = (3, 240, 240, 240, 240, 240)
-DEFAULT_DECODER_OUT_CHANNEL_DIMS = (240, 240, 240, 240, 3)
-
 
 def video_vqvae(
-    encoder_in_channel_dims: Tuple[int, ...] = DEFAULT_ENCODER_IN_CHANNEL_DIMS,
-    encoder_kernel_sizes: int = 4,
-    encoder_strides: int = 2,
+    in_channel_dim: int = 3,
+    encoder_hidden_dim: int = 240,
+    encoder_kernel_size: int = 3,
```

Comment:
You sure? Now I recall that the non-uniform kernel size for the encoder comes from VideoGPT:

https://github.com/wilson1yan/VideoGPT/blob/master/videogpt/vqvae.py#L221-L224